### PR TITLE
Bug fix/Tweak typography and spacing for browse page

### PR DESCRIPTION
**Pull request by ntsummers1**
# Title
Bug fix/Tweak typography and spacing for browse page
# Summary

This updates the browse page to more closely align with what design has mocked up. Spacing has been added, typography updated, and attention was given to the mobile to desktop transition.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How To Test

1. run `yarn dev`, then navigate to `localhost:3000`
2. See the new styles!
3. Change the browser size to see the new tablet and mobile views.

# Checklist:

- [x] My code follows the style guidelines of this project and has no linting errors
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- ~[ ] I have made corresponding changes to the documentation, including any applicable ADRs
- [x] My changes generate no new warnings
- ~[ ] I have added tests that fail without these changes
- [x] New and existing tests (unit, integration, accessibility) pass locally
- ~[ ] Documentation updated
- ~[ ] If there are security concerns they are addressed or ticketed after being discussed

### CODE REVIEW

File: src/app/components/cards/CollegeCard.tsx [hunk 1]
```diff
@@ -35,11 +35,14 @@ export const CollegeCard = ({ college }: Props) => {
...
       </CardBody>
-      <CardFooter>
+      <CardFooter className="card_footer">
         <ButtonGroup>
           <Button name="apply" type={"button"}>
             Apply to this school
           </Button>
+          <Button name="information" type={"button"} outline={true}>
+            More information
+          </Button>
```
Comment (gmrabian) [inline src/app/components/cards/CollegeCard.tsx:45]: Dan [removed this](https://github.com/coforma/swift-tech-challenge/pull/78) so just checking if we want to add it back in.

File: src/app/testing/jest/frontend/components/cards/CollegeCard.test.tsx [hunk 2]
```diff
@@ -44,8 +44,8 @@ describe("Test CollegeCard", () => {
     );
   });
 
-  test("CollegeCard button group should have one button", () => {
-    expect(screen.getAllByRole("button")).toHaveLength(1);
+  test("CollegeCard button group should have two buttons", () => {
```
Comment (TheDanMiller) [inline src/app/testing/jest/frontend/components/cards/CollegeCard.test.tsx:47]: Need to revert this test as well to one button for the time being

**Review by TheDanMiller — CHANGES_REQUESTED** [review]
See comments

File: src/app/testing/jest/frontend/components/cards/CollegeCard.test.tsx [hunk 3]
```diff
@@ -44,8 +44,8 @@ describe("Test CollegeCard", () => {
...
+    expect(screen.getAllByRole("button")).toHaveLength(2);
```
Comment (gmrabian) [inline src/app/testing/jest/frontend/components/cards/CollegeCard.test.tsx:48]: (need to update this too if we remove more info)

**Review by gmrabian — APPROVED** [review]
Looking real nice!

Just one question about code that was removed

File: src/app/components/cards/CollegeCard.tsx (hunk 1 above)
Comment (ntsummers1) [reply src/app/components/cards/CollegeCard.tsx]: Removed!

File: src/app/testing/jest/frontend/components/cards/CollegeCard.test.tsx (hunk 3 above)
Comment (ntsummers1) [reply src/app/testing/jest/frontend/components/cards/CollegeCard.test.tsx]: Fixed :D

File: src/app/components/cards/CollegeCard.tsx [hunk 4]
```diff
@@ -35,11 +35,14 @@ export const CollegeCard = ({ college }: Props) => {
           />
         </div>
       </CardBody>
-      <CardFooter>
+      <CardFooter className="card_footer">
         <ButtonGroup>
           <Button name="apply" type={"button"}>
             Apply to this school
           </Button>
+          <Button name="information" type={"button"} outline={true}>
```
[the remark this answers is not in the document]
Comment (ntsummers1) [reply src/app/components/cards/CollegeCard.tsx]: Ah gotcha. Removed again!

File: src/app/testing/jest/frontend/components/cards/CollegeCard.test.tsx (hunk 2 above)
Comment (ntsummers1) [reply src/app/testing/jest/frontend/components/cards/CollegeCard.test.tsx]: Reverted

**Review by TheDanMiller — APPROVED** [review]

**Pull request merged**